skel/django-api/deploy/stack.ts

```diff
@@ -116,20 +112,15 @@ export class ProjectNameStack extends Stack {  // FIXME change construct name
       timeout: Duration.seconds(28),
     });
 
-    const srmApi = new ApiGatewayConstruct(this, 'ApiGateway', {
-      region: this.region,
-      apiName: 'SequenceRunManager',
-      customDomainNamePrefix: 'sequence',
-      ...props,
-    });
+    const srmApi = new ApiGatewayConstruct(this, 'ApiGateway', props.apiGatewayCognitoProps);
     const httpApi = srmApi.httpApi;
 
     const apiIntegration = new HttpLambdaIntegration('ApiIntegration', apiFn);
 
-    new HttpRoute(this, 'HttpRoute', {
```

Comment:
@victorskl @reisingerf On the SRM and WFM only the `GET` method currently used?

Reply:
Read only `GET` for now.